erpnext/hr/doctype/expense_claim/expense_claim.py

```diff
@@ -127,7 +127,7 @@ def get_gl_entries(self):
 					"debit": data.sanctioned_amount,
 					"debit_in_account_currency": data.sanctioned_amount,
 					"against": self.employee,
-					"cost_center": self.cost_center
+					"cost_center": data.cost_center
```

Comment:
There is no field `cost_center` in expenses table thats why cost_center is not updated in GL Entry